README.md

```diff
@@ -127,6 +127,18 @@ This must be set before you assign an upload:
 @user.attributes = params[:user]
 ```
 
+You can also bypass backgrounding selectively only for some processors in uploader or
+only for some versions. For that an option :do_not_delay with value true should be used.
+
+```ruby
+# In an uploader
+process :set_metadata, :do_not_delay => true
```

Comment:
Seems to be that `delay: false` vs `do_not_delay` would be a cleaner solution here.